external/implementations/samlconf-openam-impl/src/main/resources/META-INF/services/org.codice.compliance.saml.plugin.IdpSSOResponder

```diff
@@ -0,0 +1 @@
+org.codice.compliance.saml.plugin.openam.OpenAMIdpSSOResponderProvider
```

Comment:
📘 This is needed to make the Implementation happy. Without this file, the OpenAM plugin isn't recognized by Kotlin and there aren't any good gradle plugins that help with this yet.

Reply:
Could we add a comment here or mention it in the README so that in the future if people want to write a plugin in Java they are aware

Reply:
I was rereading the documentation for developing a plugin and made the discovery that this isn't needed and can be replaced with the `@MetaInfServices` tag. It's documented [here](https://github.com/codice/saml-conformance#how-to-implement-a-plugin-jar) 🤦‍♂️ 

Do you think it's a good idea to show an alternate way of doing it since both solutions are described in the readme?

Reply:
No. I would prefer people use the annotation, as it's a big reason we selected that plugin for packaging these services.